src/services/item/plugins/published/service.ts

```diff
@@ -114,6 +115,36 @@ export class ItemPublishedService {
     };
   }
 
+  private async getPublishedIfExist(item: Item, itemPublishedRepository: ItemPublishedRepository) {
+    try {
+      return await itemPublishedRepository.getForItem(item);
+    } catch (_e) {
+      return undefined;
+    }
+  }
+
+  async publishIfNotExist(
+    actor: Actor,
+    repositories: Repositories,
+    itemId: string,
+    { canBePrivate }: { canBePrivate?: boolean } = {},
```

Comment:
What does that do? In which case it is ok to have the item being private?

Reply:
Yeah I copied the post method but I forgot to remove the canBePrivate params ^^